File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
 
+## Modify Footnote and Endnote content
+You can modify footnote and endnote contents in an existing Word document.
+
+The following code example shows how to modify the footnote and endnote content from an existing word document.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+//Access paragraph in Word document.
+WParagraph paragraph = document.Sections[0].Paragraphs[6] as WParagraph;
+//Get the footnote at index 0.
+WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
+//Clear footnote content.
+footnote.TextBody.ChildEntities.Clear();
+//Add new paragraph to body of footnote.
+WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
+//Set the footnote character format.
+footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append footnote text.
+footnoteParagraph.AppendText(" Footnote is modified.");
+//Access paragraph in Word document.
+paragraph = document.Sections[2].Paragraphs[1] as WParagraph;
+//Get the endnote at index 0.
```

Comment:
//Access endnote in paragraph.

Reply:
@MohanaselvamJothi - I have made correction including all the remaining platforms.